lib/winapi/filemanip.c

```diff
@@ -100,14 +101,22 @@ BOOL SetFileAttributesA (LPCSTR lpFileName, DWORD dwFileAttributes)
     return TRUE;
 }
 
+static NTSTATUS DeleteHandle(HANDLE handle)
```

Comment:
Fixed. 

I ended up adding clang-format to NevolutionX to help avoid having these types of oversights eat up reviewer time, I wonder if there's any appetite to do something similar here? Given that the nxdk is more active it might be a bit more painful since the initial fixup pass will (probably) make a lot of formatting changes that might trigger merge conflicts on outstanding PRs, but once past that it would enforce consistency.